llap-tez/src/java/org/apache/hadoop/hive/llap/tezplugins/LlapTaskSchedulerService.java

```diff
@@ -1845,7 +1845,11 @@ private static boolean removeFromRunningTaskMap(TreeMap<Integer, TreeSet<TaskInf
    */
   private boolean shouldCycle(Map<String, List<NodeInfo>> availableHostMap) {
     // short-circuit on resource availability
-    if (availableHostMap.values().stream().mapToInt(List::size).sum() > 0) return true;
+    int nodeCnt = 0;
```

Comment:
why did you change this one?
I guess the stream oneliner does the same, am I missing something?

Reply:
Functionality is the same indeed, just wanted to avoid using a stream here to avoid any performance penalties mapping Integer to Int etc.